app/components/export-basic-metadata.js

```diff
@@ -22,7 +22,7 @@ export default Component.extend({
           "resourceType": doi.types.resourceType,
           "title": doi.title,
           "author": formatCreators(doi.creators, {}),
-          "publisher": doi.publisher.name,
+          "publisher": doi.publisher ? doi.publisher.name : '',
```

Comment:
Should the blank string here be `nil`? If the DOI has no `resourceType`, is that `nil`?